artemis-core-client/src/main/resources/META-INF/services/org.apache.activemq.artemis.spi.core.remoting.ssl.SSLContextFactory

```diff
@@ -0,0 +1 @@
+org.apache.activemq.artemis.core.remoting.impl.ssl.CachingSSLContextFactory
```

Comment:
This is changing the default behavior of the broker. I'm not sure we want to do that in a minor release.